contentgrid-spring-boot-actuators/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+repositories {
+    mavenCentral()
+}
+
+configurations {
+    compileOnly {
+        extendsFrom(annotationProcessor)
+    }
+
+    testAnnotationProcessor {
+        extendsFrom(annotationProcessor)
+    }
+}
+
+dependencies {
+    annotationProcessor platform(project(':contentgrid-spring-boot-platform'))
+    annotationProcessor 'org.projectlombok:lombok'
+
+    implementation platform(project(':contentgrid-spring-boot-platform'))
+    implementation 'org.springframework.boot:spring-boot-autoconfigure'
+
+    api 'org.springframework.boot:spring-boot-actuator'
```

Comment:
`implementation` dependency ?

Reply:
or `compileOnly` ?

Reply:
we're exposing `Producible<>` from `spring-boot-actuator` here, so this needs to be `api`